optaweb-vehicle-routing-backend/pom.xml

```diff
@@ -198,6 +198,19 @@
             </excludedTestClasses>
           </configuration>
         </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-failsafe-plugin</artifactId>
+          <configuration>
+            <!--
+              Required due to the "repackage" goal of spring-boot-maven-plugin, as the maven-failsafe-plugin cannot
+              find classes in the fat jar.
```

Comment:
Could you add a link to the blog article that explained the problem and why the following "hack" solves it? I think it will be appreciated by new contributors when they come across this spot. Could also help decide whether the workaround is still needed.

Reply:
That's actually not a bad idea, will add it there.